### PR TITLE
Log project name when creating a release

### PR DESCRIPTION
**Pull request by anthonywoo**
**Note**: Samson is a public repo, do not include Zendesk-internal information, urls, etc.

Log project name when creating a release to help track which projects are using the Create Release functionality

### References
- Jira link: 

### Risks
- Low. Project should always have a name

### CODE REVIEW

File: test/controllers/integrations/base_controller_test.rb [hunk 1]
```diff
@@ -192,7 +193,7 @@ class BaseTestController < Integrations::BaseController # rubocop:disable Lint/C
         post :create, params: {test_route: true, token: token}
 
         expected_messages = <<~MESSAGES
-          INFO: Create release for branch [master], service_type [ci], service_name [base_test]: true
+          INFO: Create release for project [Foo], branch [master], service_type [ci], service_name [base_test]: true
```
Comment (rsb) [inline test/controllers/integrations/base_controller_test.rb:196]: is project [Foo] correct here

File: test/controllers/integrations/base_controller_test.rb [hunk 2]
```diff
@@ -221,7 +222,7 @@ class BaseTestController < Integrations::BaseController # rubocop:disable Lint/C
         post :create, params: {test_route: true, token: token, includes: 'status_urls'}
 
         expected_messages = <<~MESSAGES
-          INFO: Create release for branch [master], service_type [ci], service_name [base_test]: true
+          INFO: Create release for project [Foo], branch [master], service_type [ci], service_name [base_test]: true
```
Comment (rsb) [inline test/controllers/integrations/base_controller_test.rb:225]: is project [Foo] correct here

File: test/controllers/integrations/base_controller_test.rb [hunk 3]
```diff
@@ -241,7 +242,7 @@ class BaseTestController < Integrations::BaseController # rubocop:disable Lint/C
         post :create, params: {test_route: true, token: token}
 
         message = <<~MSG
-          INFO: Create release for branch [master], service_type [ci], service_name [base_test]: true
+          INFO: Create release for project [Foo], branch [master], service_type [ci], service_name [base_test]: true
```
Comment (rsb) [inline test/controllers/integrations/base_controller_test.rb:245]: is project [Foo] correct here

**Review by rsb — APPROVED** [review]

**Review by grosser — APPROVED** [review]

**Comment by grosser** [comment]
lmk if you need help with force-merge, audit fix can be it's own PR

**Pull request merged**